log/aggregator.go

```diff
@@ -0,0 +1,18 @@
+package log
+
+import (
+	"time"
+)
+
+// Aggregator is an interface for pluggable components that collect log messages delivered via
+// some transport mechanism
+type Aggregator interface {
```

Comment:
I'm 👍 with this name, but since we have a `logs` package already, I feel like `log` is confusing. can we maybe call this `transport`?